Kuroba/app/build.gradle

```diff
@@ -132,9 +123,22 @@ android {
 
 dependencies {
     implementation 'androidx.annotation:annotation:1.1.0'
-    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'androidx.browser:browser:1.0.0'
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'androidx.collection:collection:1.0.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.coordinatorlayout:coordinatorlayout:1.0.0'
+    implementation 'androidx.core:core:1.0.2'
+    implementation 'androidx.drawerlayout:drawerlayout:1.0.0'
```

Comment:
Do we even use all of this? IIRC there is a custom Drawer implementation in Clover and Kuroba.

Reply:
There was at least one file where `androidx.drawerlayout` was imported, and yeah I checked all the files for uses of AndroidX (and those were the ones I found)...

Reply:
Yeah the custom DrawerLayout extends the base implementation in AndroidX, so it's a required import. And CoordinatorLayout is used for FABs. Core of course is probably necessary for both.